### DISCUSSION TITLE
Error: Cannot find module 'quasar/dist/ssr-directives/index.js'

### DISCUSSION THREAD

**Discussion by mecjos** [quasarframework/quasar#12802] [CLI - SSR mode] [answered]
I have just created fresh quasar project.. Added `quasar mode ssr` and then run the `quasar dev -m ssr` command... then I got the following error.

Error: Cannot find module 'quasar/dist/ssr-directives/index.js'

Do i need to make some other configuration settings?? it's newly created project.

Thank you.

**Comment by rstoenescu** [comment, marked as answer]
Use the new way of scaffolding a project: "yarn create quasar" or "npm init quasar"
Also, update your global installation of "@quasar/cli" to >= 1.3.0

**Reply by Naresh-Khatri** [reply]
What are my options if the project was already created with `quasar create` command?

**Reply by rstoenescu** [reply]
> What are my options if the project was already created with `quasar create` command?
> 
> 

Same. "quasar upgrade -i"

**Reply by Naresh-Khatri** [reply]
Thank you for such a fast response and indeed that worked, you just saved my hours of research! 
Btw I have always wanted to thank you for this amazing framework and community.
